shopinvader/services/service.py

```diff
@@ -120,3 +122,25 @@ def _get_openapi_default_parameters(self):
             }
         )
         return defaults
+
+    @property
+    def shopinvader_response(self):
+        """
+        An instance of
+        ``odoo.addons.shopinvader.shopinvader_response.ShopinvaderResponse``.
+        """
+        return shopinvader_response.get()
+
+    def dispatch(self, method_name, _id=None, params=None):
+        res = super().dispatch(method_name, _id=_id, params=params)
+        store_cache = self.shopinvader_response.store_cache
+        if store_cache:
+            values = res.get("store_cache", {})
```

Comment:
Maybe we can log a message if there is a store_cache in the response and say that this way is deprecated.

Reply:
IMO not yet since we first have to clean up our own code. But it's a lot of work.